rosserial_arduino/package.xml

```diff
@@ -2,7 +2,7 @@
   <name>rosserial_arduino</name>
   <version>0.7.6</version>
   <description>
-    Libraries and examples for ROSserial usage on Arduino/AVR Platforms.
+    ROSserial for Arduino/AVR Platforms.
```

Comment:
Elsewhere, the name is styled as `rosserial`, all lowercase. Please use that here, since we're updating.